design/template.md

```diff
@@ -0,0 +1,232 @@
+This template is adapted from Kubernetes Enchancements KEP template https://raw.githubusercontent.com/kubernetes/enhancements/a86942e8ba802d0035ec7d4a9c992f03bca7dce9/keps/NNNN-kep-template/README.md
+
+# Title
```

Comment:
suggestion: I think this is the only title we'd _always_ want people to change; maybe we can make it clearer that the user should change it?

```suggestion
# CHANGEME: Title
```